mxcube3/ui/components/SampleView/ContextMenu.js

```diff
@@ -99,11 +100,27 @@ export default class ContextMenu extends React.Component {
   showModal(modalName, wf = {}, _shape = null) {
     const { sampleID, defaultParameters, shape, sampleData } = this.props;
     const sid = _shape ? _shape.id : shape.id;
+    const cellCount = shape.gridData ? defaultParameters[modalName.toLowerCase()].cell_counting
+      : 'None';
+
+    let aux;
```

Comment:
So aux is actually num_images ?

Reply:
Do we still need to pass numCols or numRows as number of images ?

Reply:
yes, because num_images is the value that will go to the acquisition parameters. It is removed from the modal display (substituted by total_num_images) but still needed

Reply:
I think that num_images should always be num_images, if we need to pass num cols and num rows we should pass them as num_cols and num_rows and use them where needed.

num_images should always be total number of images